src/asm/globlex.c

```diff
@@ -100,7 +100,10 @@ static int32_t ascii2bin(char *s)
 		break;
 	}
 
-	if (radix == 4) {
+	if (radix != 10 && *s == '\0') {
```

Comment:
Do we really need the "radix != 10"? As far as I see, the only way to have radix = 10 is to have a decimal number (with no prefix), so the only way to get an empty number with radix = 10 is a serious parser problem that shouldn't happen (passing an empty string to this function).

Can't you remove it?

Reply:
I agree. It's not necessary.